<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1070</width>
    <height>712</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="BackButton">
   <property name="geometry">
    <rect>
     <x>900</x>
     <y>640</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_base">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>40</y>
     <width>851</width>
     <height>511</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="WindowOn">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>610</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>WindowOn</string>
   </property>
  </widget>
  <widget class="QPushButton" name="lightOn">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>570</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>lightOn</string>
   </property>
  </widget>
  <widget class="QPushButton" name="WindowOff">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>610</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>WindowOff</string>
   </property>
  </widget>
  <widget class="QPushButton" name="DoorOff">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>660</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>DoorOff</string>
   </property>
  </widget>
  <widget class="QPushButton" name="DoorOn">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>660</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>DoorOn</string>
   </property>
  </widget>
  <widget class="QPushButton" name="lightOff">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>570</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>lightOff</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_light">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>40</y>
     <width>221</width>
     <height>511</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignJustify|Qt::AlignVCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
